actionpack/lib/action_dispatch/middleware/params_parser.rb

```diff
@@ -28,16 +39,16 @@ def initialize(app, parsers = {})
     def call(env)
       request = Request.new(env)
 
-      request.request_parameters = parse_formatted_parameters(request, @parsers)
+      request.request_parameters = parse_formatted_parameters(request)
```

Comment:
Could you undo this change? We are passing the instance variable by design. See https://github.com/rails/rails/commit/f3bae24c81e7198d9bbedaf79056d9961e59113c